src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -100,6 +104,51 @@ public void removePerson(Person key) {
         indicateModified();
     }
 
+    // ==== Added methods to support operations on UniqueHealthWorkerList ====
+    // @author: Lookaz
+
+    /**
+     * Returns true if a HealthWorker with the same identity as {@code worker}
+     * exists in the address book.
+     */
+    public boolean hasHealthWorker(HealthWorker worker) {
```

Comment:
Just to clarify, do these methods currently check the JSON file for `HealthWorker` or do they require implementation of the `Storage` component first?

Reply:
As of now they do not. The current implementations for operations supporting HealthWorker objects only deal with HealthWorker objects in the internal list.